docs/schemaActions/README.md

```diff
@@ -1,27 +1,101 @@
 
 In both cases a value of true is assumed so the element only needs to be included if the action was unsuccessful.
 
-The `<Description>` element should be used to described the detail around the failure of the action as this will be useful to a human looking at the event.
+The `<Description>` element should be used to described the detail around the failure of the action as this
+ will be useful to a human looking at the event.
 
 ### Data
-This provides a flexible structure for describing aspects of the schema action that cannot be described using the structure in the schema. This structure is described in more detail in [Unstructured Data](../unstructuredData.md)
+This provides a flexible structure for describing aspects of the schema action that cannot be described using
+ the structure in the schema. This structure is described in more detail in [Unstructured Data](../unstructuredData.md)
+
+The majority of the 'schema action' elements include the MultiObjectComplexType structure (or some child of it)
+ to allow for recording various forms of objects that were involved in the event.
+
+##Defined Schema Actions
+A number of schema actions are defined that aim to be broadly representative of the kinds of activity that are 
+most likely to be of interest from an audit point of view.
+
+Schema Action|Rough Description |
+|---|---|
+| [Alert](alert.md) | A potentially concerning situation has been identified *that requires user attention to resolve*.|
+| [AntiMalware](antiMalware.md) | Events relating to functioning of antimalware softwrae *Will be removed in a future version of the schema*.|
+| [Authorise](authorise.md) | Events relating to authorisation changes, e.g. to group membership within LDAP or AD.|
+| [Copy](copyMove.md) | Making copies of data. |
+| [Create](createViewDelete.md) | Creating new items of data. |
+| [Delete](createViewDelete.md) | Destroying data. |
+| [Export](importExport.md) | Moving data out of a controlled area (e.g. database, application or network). |
+| [Import](importExport.md) | Moving data into a controlled area (e.g. database, application or network). |
+| [Install](installUninstall.md) | Installing hardware, software or removable media.|
+| [Move](copyMove.md) | Moving data. |
+| [Network](network.md) | Events that relate to networking between computers. |
+| [Print](printing.md) | Events relating to printing (i.e. making hard-copies). |
+| [Process](process.md) | Generic processing events, including starting processes and services on computers. |
+| [Receive](sendReceive.md) | Obtaining data over the network. |
+| [Search](search.md) | Search operations, e.g. querying a database. |
+| [Send](sendReceive.md) | Transmitting data over the network. |
+| [Uninstall](installUninstall.md) | Events relating to removal of hardware, software or removable media.|
+| [Unknown](unknown.md) | A type of event that is dissimilar to any in the schema. |
```

Comment:
Should only be used in niche cases, if used a lot it implies that a new schema action may be required.

Reply:
added in #26